commands/firstline/index.js

```diff
@@ -3,14 +3,14 @@ module.exports = {
 	Aliases: ["fl"],
 	Author: "supinic",
 	Cooldown: 5000,
-	Description: "Posts the target user's first chat line in the context of the current channel, and the date they sent it.",
+	Description: "Posts the target user's first chat line in the context of the current or a specified channel, and the date they sent it.",
 	Flags: ["external-input","mention","opt-out","pipe"],
 	Params: [
 		{ name: "textOnly", type: "boolean" }
 	],
 	Whitelist_Response: null,
 	Static_Data: null,
-	Code: (async function firstLine (context, user) {
+	Code: (async function firstLine (context, user, channel) {
```

Comment:
Based on discussion, move the `channel` argument to command `Params` - making sure to not "disturb" the original functionality.